test-cli.py

```diff
@@ -420,7 +422,7 @@ def test(ctx):
     check_contains(['run "echo hello"'], run_command([cl, 'info', '-f', 'args', uuid]))
     check_equals('hello', run_command([cl, 'cat', uuid+'/stdout']))
     # block
-    uuid2 = check_contains('hello', run_command([cl, 'run', 'echo hello', '--tail'])).split('\n')[0]
+    uuid2 = check_contains('hello', run_command([cl, 'run', 'echo hello', '--tail']))
```

Comment:
It seems like uuid2 isn't being used at all, so I think I was actually going to go ahead and delete the assignment as well but forgot about it. Would you rather I leave it in?